geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/sortedset/AbstractZInterStoreIntegrationTest.java

```diff
@@ -78,6 +78,32 @@ public void shouldError_givenWrongKeyType() {
         STRING_KEY, KEY1)).hasMessage("WRONGTYPE " + RedisConstants.ERROR_WRONG_TYPE);
   }
 
+  @Test
+  public void shouldError_givenTwoSortedSetKeysAndWrongKeyType() {
+    Map<String, Double> scores = buildMapOfMembersAndScores(1, 5);
+    Map<String, Double> nonIntersectionScores = buildMapOfMembersAndScores(6, 10);
+    jedis.zadd(KEY1, scores);
+    jedis.zadd(KEY2, nonIntersectionScores);
+
+    final String STRING_KEY = "{tag1}stringKey";
```

Comment:
Minor point, but all-caps variable names should be reserved for static constants rather than local variables. (See the [Google Java Style guide](https://google.github.io/styleguide/javaguide.html#s5.2-specific-identifier-names) that Geode follows). I think the reason there are some local variables with the wrong format in this file already is due to previous refactoring when they were turned from constants into local variables but not renamed.

Reply:
Should I rename the other local variables in the class to follow that format.

Reply:
That would be awesome, thanks!